alibi_detect/utils/tensorflow/distance.py

```diff
@@ -198,7 +198,7 @@ def permed_lsdds(
 
     Returns
     -------
-    Vector of B LSDD estimates for each permutation, H_lam_inv which may have been inferred, and optionally
+    Vector of B LSDD estimates for each permutation, H_lam_inv which may have been inferred, and optionally \
```

Comment:
Are these explicit line breaks actually needed?

Reply:
Didn't realise this, good spot @mauicv !